test/controllers/stripe_connect_events_controller_test.exs

```diff
@@ -94,9 +92,38 @@ defmodule CodeCorps.StripeConnectEventsControllerTest do
 
       assert response(conn, 200)
 
-      updated_project =
-        Project
-        |> Repo.get_by(id: project.id)
+      updated_project = Project |> Repo.get_by(id: project.id)
+
+      assert updated_project.total_monthly_donated == 0
+    end
+  end
+
+  describe "customer.subscription.deleted" do
+    test "returns 200 and sets subscription to inactive when one matches", %{conn: conn} do
+      event = event_for(@subscription, "customer.subscription.deleted")
+      stripe_id =  @subscription["id"]
+      connect_customer_id = @subscription["customer"]
+
+      project = insert(:project, total_monthly_donated: 1000)
+      account = insert(:stripe_connect_account)
+      platform_customer = insert(:stripe_platform_customer)
+
+      insert(:stripe_connect_customer,
+        id_from_stripe: connect_customer_id,
+        stripe_connect_account: account,
+        stripe_platform_customer: platform_customer)
+
+      plan = insert(:stripe_connect_plan, project: project)
+
+      insert(:stripe_connect_subscription,
+        id_from_stripe: stripe_id,
+        stripe_connect_plan: plan)
+
+      conn = post conn, stripe_connect_events_path(conn, :create), event
+
+      assert response(conn, 200)
```

Comment:
Thoughts on piping this block?

`assert conn |> post(stripe_connect_events_path(conn, :create), event) |> response(200)`